tests/functional/ca/build-cache.sh

```diff
@@ -26,7 +26,9 @@ copyAttr () {
     # Note: to copy CA derivations, we need to copy the realisations, which
     # currently requires naming the installables, not just the derivation output
     # path.
-    nix copy --to file://$cacheDir "${args[@]}"
+
+    # shellcheck disable=SC2154
```

Comment:
I am a bit surprised by this because args is defined above

Reply:
Fixed now, See the `.shellcheck` change